plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java

```diff
@@ -1265,6 +1291,30 @@ private void initializeFromMetadata() {
             }
         }
 
+        // scales and offsets, if any
+        this.offsets = null;
+        this.scales = null;
+        f = imageMetadata.getTIFFField(PrivateTIFFTagSet.TAG_GDAL_METADATA);
+        this.offsets = null;
+        this.scales = null;
```

Comment:
duplicate initialization?
        this.offsets = null;
        this.scales = null;

Reply:
Meglio abbondare!
(I'll fix it :-p )